examples/getReleaseAsset.js

```diff
@@ -4,40 +4,41 @@ const github = new GitHubApi({
   debug: true
 })
 
-var testRepo = {
+github.repos.getReleases({
   owner: 'octokit',
   repo: 'node-github'
-}
+})
 
-github.repos.getReleases({
-  owner: testRepo.owner,
-  repo: testRepo.repo
-}, function (err, res) {
-  if (err) throw err
-  var releases = res.data
-  if (releases.length === 0) {
+.then(result => {
+  if (result.data.length === 0) {
+    console.log('Repository has no releases')
     return
   }
-  var release = releases[0]
-  var releaseId = release.id
 
-  github.repos.getAssets({
-    owner: testRepo.owner,
-    repo: testRepo.repo,
-    id: releaseId
-  }, function (err, res) {
-    if (err) throw err
-    var assets = res.data
-    if (assets.length === 0) {
-      return
-    }
-    var asset = assets[0]
-    var assetId = asset.id
+  // get id of first release
+  const firstRelease = result.data.pop()
+  return github.repos.getAssets({
+    owner: 'octokit',
+    repo: 'node-github',
+    id: firstRelease.id
+  })
+})
+
+.then(result => {
+  if (result.data.length === 0) {
+    console.log('First release has no assets attached')
+    return
+  }
 
-    github.repos.getAsset({
-      owner: testRepo.owner,
-      repo: testRepo.repo,
-      id: assetId
-    })
+  // get id of first asset
+  const assetId = result.data[0].id
+  return github.repos.getAsset({
+    owner: 'octokit',
+    repo: 'node-github',
+    id: assetId
   })
 })
+
+.then(result => {
+  // result.data has asset properties
+})
```

Comment:
According to the code coverage tool we are missing a big chunk on this file: ... 29, 30, 34, 35. I believe the statement on line 12 is throwing an error which is skipping the remaining .then calls.